app/src/main/res/values/strings.xml

```diff
@@ -3,45 +3,45 @@
     <string name="app_name" translatable="false">Seedvault</string>
     <string name="backup">Backup</string>
     <string name="data_management_label">Seedvault Backup</string>
-    <string name="current_destination_string">Backup status and settings</string>
+    <string name="current_destination_string">Back up status and settings</string>
 
     <string name="restore_backup_button">Restore backup</string>
 
     <!-- Settings -->
     <string name="settings_category_apps">App backup</string>
-    <string name="settings_backup">Backup my apps</string>
```

Comment:
This is for sure wrong, which is my confidence level for changing other such instances.